examples/with-tailwind/tailwind.config.js

```diff
@@ -0,0 +1,18 @@
+const colors = require("tailwindcss/colors");
+const path = require("path");
+
+module.exports = {
+  content: [
+    `${path.resolve(__dirname, "apps/")}/**/*.{js,ts,jsx,tsx}`,
+    `${path.resolve(__dirname, "packages/")}/**/*.{js,ts,jsx,tsx}`,
```

Comment:
This, where nothing in packages appears to be using the tailwind setup, makes for an unappealing configuration line that I'd love to remove.

Reply:
This is the last remaining issue, as we discussed offline - we have to enumerate where all of the tailwind css classes are being used: https://tailwindcss.com/docs/content-configuration#configuring-source-paths. For packages that include css as well, we have to enumerate those as well https://tailwindcss.com/docs/content-configuration#working-with-third-party-libraries.

This gets tricky in a monorepo, because in this case, the css for packages/ui will always be included by every app that uses this config, even if the app isn't using the ui package. 

We can fix this by bundling the ui package and importing the css independently of the app build.